README.md

```diff
@@ -157,24 +152,8 @@ In settings.py,
         # Set custom model for attachments (default: 'django_summernote.Attachment')
         'attachment_model': 'my.custom.attachment.model', # must inherit 'django_summernote.AbstractAttachment'
 
-        # Set common css/js media files
-        'base_css': (
-            '//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css',
-        ),
-        'base_js': (
-            '//code.jquery.com/jquery-1.9.1.min.js',
-            '//netdna.bootstrapcdn.com/bootstrap/3.1.1/js/bootstrap.min.js',
-        ),
-        'default_css': (
-            os.path.join(STATIC_URL, 'django_summernote/summernote.css'),
-            os.path.join(STATIC_URL, 'django_summernote/django_summernote.css'),
-        ),
-        'default_js': (
-            os.path.join(STATIC_URL, 'django_summernote/jquery.ui.widget.js'),
-            os.path.join(STATIC_URL, 'django_summernote/jquery.iframe-transport.js'),
-            os.path.join(STATIC_URL, 'django_summernote/jquery.fileupload.js'),
-            os.path.join(STATIC_URL, 'django_summernote/summernote.min.js'),
-        ),
+        # You can disable attachment feature.
+        'disable_attachment': False,
```

Comment:
The issue I have with this is that you still need to put in two or more settings to disable uploads properly `'disable_attachment': True` and `'summernote': { 'disableDragAndDrop': True }` as well as remove the toolbar button appropriately. 

Is this by design, to allow for inlining image uploads as `data` values in the widget output? If so, then ignore my comment! 🤔💥

See my comment in my pull at https://github.com/summernote/django-summernote/pull/279#issuecomment-383211026

Reply:
I agree that these options are pretty easy to be misunderstood. Yes, it caused by Summernote design. 

Summernote is designed to attach images as base64-encoded, but I think most of Django users need file-based attachment system like how we've done before. So I added `Attachment` model in `django-summernote` and also added a way to customize image uploading in `summernote`.

Thus, `disableDragAndDrop` only disables drag-and-drop feature of `Summernote` and `disable_attachment` only disable `attachment` feature of `django-summernote`.

If we want to add an option for disabling the whole features of image uploading, it must be `disable_image` or something else, not `disable_attachment`.

Reply:
Understood, that history lesson makes sense - thanks for taking the time to explain it. I think what you've got makes sense then.